modules/ipi-install-diagnosing-duplicate-mac-address.adoc

```diff
@@ -0,0 +1,48 @@
+[id="ipi-install-diagnosing-duplicate-mac-address_{context}"]
+= Diagnosing a duplicate MAC address when provisioning a new host in the cluster
+
+[role="_abstract"]
```

Comment:
Despite organizational guidelines to add this attribute, the team decision at the moment is to exclude it.  Please remove the attribute.